tests/test_fuzzy_dedup.py

```diff
@@ -71,6 +71,24 @@ def large_fuzzy_dedup_data():
     return DocumentDataset(df)
 
 
+@pytest.fixture
+def shuffle_fail_fuzzy_dedup_data():
+    df = cudf.DataFrame(
+        {
+            "id": [1, 2, 300, 4, -1],
+            "text": [
+                "A test string",
+                "A different test string",
+                "A different object",
+                "Something completely else that doesn't match",
+                "The quick black cat jumps over the lazy dog",
```

Comment:
Added this test where there is only a group of first three documents (at a higher jaccard similarity only first two are grouped)